lib/src/builder/codegen/accessors_generator.dart

```diff
@@ -65,15 +65,22 @@ abstract class TypedMapAccessorsGenerator extends BoilerplateDeclarationGenerato
     final typeParamsOnClass = typeParameters?.toSource() ?? '';
     final typeParamsOnSuper = removeBoundsFromTypeParameters(typeParameters);
 
-    final deprecationLine = internalGeneratedMemberDeprecationLine(
+    final generatedCodeUseOnlyDeprecation = internalGeneratedMemberDeprecationLine(
```

Comment:
Renamed for clarity since this doesn't do anything with user-authored deprecation annotations.